src/TesApi.Web/BatchScheduler.cs

```diff
@@ -38,6 +38,7 @@ public class BatchScheduler : IBatchScheduler
         private const int DefaultMemoryGb = 2;
         private const int DefaultDiskGb = 10;
         private const string CromwellPathPrefix = "/cromwell-executions/";
+        private const string ExecutionsPathPrefix = "/executions";
```

Comment:
maybe this should end with "/" to be consistent with the prefix above it?  then would need to change the way it's used in GetBatchExecutionDirectoryPath too